### PR TITLE
Table for search queries

### PR DESCRIPTION
**Pull request by JaabaShooler**
image
Added a "Search Queries Report" page, created table component
image

### CODE REVIEW

File: src/components/SearchQueries/SearchQueries.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import './SearchQueries.scss';
+import useSWR from 'swr';
+import Table from '../Tools/Table/Table';
+import {fetchSearchQueries} from '../../helpers/api';
+
+const SearchQueries = () => {
+  const {data: searchQueries} = useSWR('get-search-queries', fetchSearchQueries);
+
+  console.log(searchQueries);
```
Comment (MariannaMilovanova) [inline src/components/SearchQueries/SearchQueries.tsx:10]: console.log better to to comit

File: src/components/Tools/Table/Table.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+const Table: React.FC<ITableProps> = ({edges}) => {
+  return (
+    <div className="Table">
+      <div className="Table__wrapper">
+        <div className="Table__column">
+          <label className="Table__element Table__header">#</label>
+          <label className="Table__element Table__header">Query</label>
+          <label className="Table__element Table__header right">Total Searches</label>
+          <label className="Table__element Table__header right">Broad Searches</label>
```
Comment (pihorb) [inline src/components/Tools/Table/Table.tsx:23]:
```suggestion
          <label className="Table__element Table__header--right">Broad Searches</label>
```

File: src/components/Tools/Table/Table.tsx [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+          <label className="Table__element Table__header right">Exact Searches</label>
+        </div>
+        {edges.map((query, index) => (
+          <div className="Table__column" key={query.node.text}>
+            <label className="Table__element">{index + 1}.</label>
+            <label className="Table__element">{query.node.text}</label>
+            <label className="Table__element right">{query.totalCount}</label>
```
Comment (pihorb) [inline src/components/Tools/Table/Table.tsx:30]:
```suggestion
            <label className="Table__element Table__element--right">{query.totalCount}</label>
```

**Review by pihorb — APPROVED** [review]
GTG after fixing conflicts

**Review by hmelenok — APPROVED** [review]

**Pull request merged**